### ISSUE TITLE
Confusion over WP_RIG installation process

### ISSUE THREAD

**Issue by kekir** [wprig/wprig#664]
## Issue Overview
I am trying to install wp-rig, but i am confused as there are 3 different set of instructions provided, here https://wprig.io/#install, here https://github.com/wprig/wprig/ and one in form of video https://wprig.io/learn/. I am not able to follow either of them as the git-hub repo that we are told to use does not match that in the instruction and video, to be precise it does not have a dev folder as they say it should. Could you please be precise what can be done in the said regards. thanks 

## Steps to Reproduce

1.
2.
3.
4.

**Comment by alexwcoleman** [comment]
Hi! Sorry for the delay regarding this. Please try the [How To Install instructions in this repo](https://github.com/wprig/wprig#how-to-install-wp-rig). The video you watched sounds like the instructions for an older version.

**Comment by alexwcoleman** [comment]
I've also updated the website's front page. Please drop a comment if there is still confusion.

**Comment by madebyvale** [comment]
Hi @alexwcoleman thanks for the info provided, this helps.

**Comment by kekir** [comment]
Hi Alex,
thanks for getting back , but,
I wasted a lot of time installing and resolving issues that wp-rig poses,
the documentation is not full proof and indicative of its outcome either.
Bits and pieces of information are scattered everywhere, Which makes it
difficult for an intermediate developer like me to figure out things. I
also browsed a lot of issues that were being posted and not replied to,
upon digging further, I also found out that wp-rig is not maintained for
the last year or so. So I am not sure whether the project is dead or will
be there to support its users.
thanks keki
